.github/workflows/build-publish.yaml

```diff
@@ -28,6 +28,19 @@ jobs:
   build:
     name: Build image
     runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        include:
+          - containerfile: "./chat/base/Containerfile"
+            build-image-name: "ai-lab-playground-chat"
+            archs: amd64, arm64
+          - containerfile: "./chat/vulkan/amd64/Containerfile"
+            build-image-name: "ai-lab-playground-chat-vulkan-amd64"
```

Comment:
The image name should not contain the arch so this should be handled through a manifest

Reply:
there are two versions of the same image. One for amd and one for arm. We should use the one we need based on the system

Reply:
Yes but we should select based on the arch not the name just like the base image (CPU)

Reply:
Removed the arch from the name. is that enough?

Reply:
Ops, updated the other file :face-palm:

Reply:
Updated